homeassistant/components/rainmachine/config_flow.py

```diff
@@ -29,6 +29,9 @@
     DOMAIN,
 )
 
+API_URL_PLACEHOLDERS = {
+    "api_url": "https://rainmachine.docs.apiary.io/#reference/weather-services/parserdata/post",
+            }
```

Comment:
```suggestion
```

The translation strings with an URL belong to a service action, and are not related to a data entry flow.

Reply:
Add the place holder here:

https://github.com/home-assistant/core/blob/973405822b1e8467c58e99a1c8da8ec6365a4324/homeassistant/components/rainmachine/__init__.py#L458